spec/models/concerns/searchable_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'spec_helper'
+
+RSpec.describe KB::Searchable do
```

Comment:
This spec if fine, but it seems to be focused on an implementation details. I'd much rather prefer a test for an actual usage of this module in the `PetContract` model. I like how it was done with referrals: https://github.com/barkibu/kb-ruby/pull/74

There are two tests: [one](https://github.com/barkibu/kb-ruby/pull/74/files#diff-631f1e22871256d30e41df92b56b56824cad66312b62b1a5c1cfd5edf90fb34f) for fake memory datasource and [another](https://github.com/barkibu/kb-ruby/pull/74/files#diff-631f1e22871256d30e41df92b56b56824cad66312b62b1a5c1cfd5edf90fb34f) with remote API stubbed via webmock.